ui/src/app/components/update-spanner-config-form/update-spanner-config-form.component.html

```diff
@@ -0,0 +1,27 @@
+<div mat-dialog-content>
```

Comment:
Are we not following the figma for this UI component ?
https://www.figma.com/file/mhSfts8ey0XxeHXyXdP1aC/HarbourBridge-UI?node-id=97%3A60

Reply:
We will be updating the form to match with Figma. As the Figma design for this form was not there initially we used a dialog form. So this change goes to backlog.